integrations/openai/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@botpress/openai",
+  "integrationName": "openai",
+  "main": "./src/index.ts",
+  "scripts": {
+    "start": "pnpm bp serve",
+    "build": "pnpm bp build",
+    "deploy": "pnpm bp deploy",
+    "type-check": "tsc --noEmit",
+    "test": "echo \"Tests not implemented yet.\""
```

Comment:
**chore:** rm this unused script

Reply:
Just realized the GH actions for integrations don't call a "test" npm script as they did before, will remove.